contracts/PartnerConfiguration/PartnerConfiguration.sol

```diff
@@ -299,8 +299,8 @@ contract PartnerConfiguration is IPartnerConfiguration, Ownable {
        @inheritdoc IPartnerConfiguration
      */
     function getPrice(
-        string calldata /* name */,
```

Comment:
If we remove this, wouldn't we be changing the required interface?

Reply:
the oposite. it is actually implementing the interface. Somebody removed the params to silence the warnings. But the docs werent working because that function didnt have the names of the params. so I just added the missing param names.